packages/server/src/fhir/job.ts

```diff
@@ -33,6 +36,48 @@ jobRouter.get(
   })
 );
 
-jobRouter.delete('/:id/status', (req: Request, res: Response) => {
-  res.sendStatus(202);
-});
+jobRouter.delete(
+  '/:id/status',
+  asyncWrap(async (req: Request, res: Response) => {
+    let normalizedOutcome: OperationOutcome;
+    try {
+      const request: FhirRequest = {
+        method: req.method as HttpMethod,
+        url: req.originalUrl.replace('/fhir/R4', ''),
+        pathname: '',
+        params: req.params,
+        query: {},
+        body: req.body,
+        headers: req.headers,
+      };
+      const [outcome] = await asyncJobCancelHandler(request);
```

Comment:
We call `asyncJobCancelHandler` for this request and look at the outcome. The only special case is that we want `allOk` to translate to a 202 and an "accepted" `OperationOutcome`